.travis.yml

```diff
@@ -1,18 +1,24 @@
 language: python
 python:
-  - "2.7"
```

Comment:
I remember there was a reason that the Viderum guys built this on 2.x.
There may be some incompatibility in the CKAN python libraries. Just something to watch out for.

Reply:
https://www.bleepingcomputer.com/news/software/python-27-reaches-end-of-life-after-20-years-of-development/

Reply:
In the README says python 2.7  :nerd_face: 

> ### Running the app
> The project is setup with Docker. It will spin up an environment with Python **2.7** based on Alpine and install the required dependencies.

Reply:
@lordmallam I assume this is all ok.